### PR TITLE
runtests.1: rephrase how to specify what tests to run

### PR DESCRIPTION
**Pull request by bagder**
Also mention the new minus-prefixed way to ignore test results.

### CODE REVIEW

File: tests/runtests.1 [hunk 1]
```diff
@@ -20,28 +20,36 @@
...
+
+If no test number or keyword is given, all existing tests that the script can
+find will be considered for running. You can specify single test cases to run
+by specifying test numbers space-separated, like "1 3 5 7 11", and you can
+specify a range of tests like "45 to 67".
+
+Specify tests to not run with a leading exclamation point, like "!66", which
+runs all available tests except number 66.
+
+Prefix a test number with a minus (-) to still run it, but to ignore the
```
Comment (mback2k) [inline tests/runtests.1]: Actually it is not a minus (-), but a tilde (~) character as prefix.

File: tests/runtests.1 [hunk 2]
```diff
@@ -20,28 +20,36 @@
...
+results.
+
+It is also possible to specify tests based on a keyword describing the test(s)
+to run, like "FTPS". The keywords are strings used in the indiviual tests.
+
+You can also specify keywords with a leading exclamation point and the keyword
+or phrase, like "!HTTP NTLM auth" to run all tests \fBexcept\fP those using
```
Comment (mback2k) [inline tests/runtests.1]: The ignoring of results using tilde (~) also works with keywords, just like !

**Comment by mback2k** [comment]
This PR is a partial follow-up to #4994.

**Comment by bagder** [comment]
Updated and forced-pushed

**Pull request closed without merging**